book/snippets.rst

```diff
@@ -0,0 +1,74 @@
+Using Snippets
```

Comment:
would use this file as `SnippetBundle` documentation (similar to `LocationBundle` or `RouteBundle`)